### PR TITLE
tests: Add docker inspect test

### LINKED ISSUE #218: add docker inspect test for integration tests
Add docker inspect test for our integration tests

### PR DESCRIPTION
**Pull request by GabyCT**
This will test the docker inspect option as part of the integration tests

Fixes #218

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
lgtm

File: integration/docker/inspect_test.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+		args = []string{"run", "-t", "--name", id, Image, "true"}
+		command := NewCommand(Docker, args...)
+		Expect(command).ToNot(BeNil())
+		exitCode := command.Run()
+		Expect(exitCode).To(Equal(0))
+
+	})
+
+	AfterEach(func() {
+		args := []string{"rm", "-f", id}
```
Comment (devimc) [inline integration/docker/inspect_test.go]: we already have a method to remove a container
https://github.com/clearcontainers/tests/blob/master/docker.go#L63

File: integration/docker/inspect_test.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	DescribeTable("inspect with docker",
+		func(formatOption string) {
+			args = []string{"inspect", id, "--format"}
+			args = append(args, formatOption)
+
+			command := NewCommand(Docker, args...)
+			exitCode := command.Run()
+			Expect(exitCode).To(Equal(0))
+		},
+		inspectFormatOptions("'{{.Config.Image}}'"),
```
Comment (devimc) [inline integration/docker/inspect_test.go]: would be nice if you could inspect the container id and verify that it is the same

File: integration/docker/inspect_test.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+		inspectFormatOptions("'{{json .Config}}'"),
+		inspectFormatOptions("'{{.LogPath}}'"),
```
Comment (devimc) [inline integration/docker/inspect_test.go]: and verify that the path exist or something like that, but you could do this in a next PR

**Comment by GabyCT** [comment]
@devimc changes applied thanks for the feedback

**Comment by devimc** [comment]
lgtm

**Pull request merged**